rest/admin_api_test.go

```diff
@@ -801,59 +801,51 @@ func TestSessionTtlGreaterThan30Days(t *testing.T) {
 }
 
 func TestSessionExtension(t *testing.T) {
-
-	if testing.Short() {
-		t.Skip("skipping test in short mode")
-	}
-
 	rt := NewRestTester(t, nil)
 	defer rt.Close()
 
 	a := auth.NewAuthenticator(rt.Bucket(), nil)
-	user, err := a.GetUser("")
-	assert.NoError(t, err)
-	user.SetDisabled(true)
-	err = a.Save(user)
-	assert.NoError(t, err)
-
-	user, err = a.GetUser("")
-	assert.NoError(t, err)
-	goassert.True(t, user.Disabled())
-
-	response := rt.SendRequest("PUT", "/db/doc", `{"hi": "there"}`)
-	assertStatus(t, response, 401)
-
-	user, err = a.NewUser("pupshaw", "letmein", channels.SetOf(t, "*"))
+	user, err := a.NewUser("pupshaw", "letmein", channels.SetOf(t, "*"))
 	a.Save(user)
 
-	assertStatus(t, rt.SendAdminRequest("GET", "/db/_session", ""), 200)
+	if err != nil {
+		log.Printf(err.Error())
+	}
 
-	response = rt.SendAdminRequest("POST", "/db/_session", `{"name":"pupshaw", "ttl":1}`)
-	assertStatus(t, response, 200)
+	// More than 10% of expiry (24h)
+	fakeSession := auth.LoginSession{
+		ID:         base.GenerateRandomSecret(),
+		Username:   "user1",
+		Expiration: time.Now().Add(time.Hour * 4),
+		Ttl:        time.Hour * 24,
+	}
 
-	var body db.Body
-	base.JSONUnmarshal(response.Body.Bytes(), &body)
-	sessionId := body["session_id"].(string)
-	sessionExpiration := body["expires"].(string)
-	assert.NotEmpty(t, sessionId)
-	assert.NotEmpty(t, sessionExpiration)
-	assert.True(t, body["cookie_name"].(string) == "SyncGatewaySession")
+	rt.Bucket().Set(docIDForSession(fakeSession.ID), 0, fakeSession)
```

Comment:
`Set` returns an error. Let's wrap this in an `assert.NoError` just to be safe.

```suggestion
	assert.NoError(t, rt.Bucket().Set(docIDForSession(fakeSession.ID), 0, fakeSession))
```

Reply:
This change has been committed

Reply:
This change has been committed.